.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''

```

Comment:
does it make sense to create and add the "feature" label here.  
Is there any easy way to tell the difference when looking at the issues?